src/constants/config.js

```diff
@@ -4,6 +4,7 @@ const CONFIG = {
   PORT: process.env.PORT,
   MONGODB_URL: process.env.MONGODB_URL,
   CLIENT_URL: process.env.CLIENT_URL,
+  SECRET_KEY: process.env.SECRET_KEY,
```

Comment:
아래 쿠키 maxAge 상수화 처리와 연관된 제안사항입니다.

```suggestion
  PORT: process.env.PORT,
  MONGODB_URL: process.env.MONGODB_URL,
  CLIENT_URL: process.env.CLIENT_URL,
  SECRET_KEY: process.env.SECRET_KEY,
  ONE_HOUR_IN_MS: 1000 * 60 * 60;
```